app.py

```
@@ -1,29 +1,41 @@
+                cur = mysql.connection.cursor()
+                cur.execute("""
+                    CREATE TABLE IF NOT EXISTS messages (
+                        id INT AUTO_INCREMENT PRIMARY KEY,
+                        message TEXT
+                    );
+                """)
+                mysql.connection.commit()
+                cur.close()
+                print("✅ Database initialized")
+                return
+        except OperationalError as e:
```

Ruff (0.14.10)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)